tests/aeroval/test_aeroval_config_diurnal.py

```diff
@@ -0,0 +1,104 @@
+import os.path
+import pathlib
+
+import pyaro
+
+from pyaerocom.aeroval import EvalSetup, ExperimentProcessor
+from pyaerocom.aeroval.config.ciconfigs.base_config import get_CFG
+from pyaerocom.io import ReadUngridded
+from pyaerocom.io import ReadGridded
+
+reportyear = year = 2018
+CFG = get_CFG(
+    reportyear=reportyear,
+    year=year,
+)
+
+TEST_FILE = "mep-rd-Birkenes-2018-001.nc"
+
+
+def test_harp_reader_available():
```

Comment:
These tests should be skipped if pyaro-readers is not installed

Reply:
That might lead false positive test runs that make developers think that everything is alright while it isn't. Let's discuss this at the dev meeting.

Reply:
I too would like to see pyaro-readers being included in pyaerocom by default. Lets discuss adding it to the optional dependencies under `pyproject.toml`/`test`